src/Microsoft.OData.Client/QueryOperationResponseOfT.cs

```diff
@@ -42,6 +42,7 @@ internal QueryOperationResponse(HeaderCollection headers, DataServiceRequest que
 
         /// <summary>The server result set count value from a query, if the query has requested the value.</summary>
         /// <returns>The return value can be either zero or a positive value equal to the number of entities in the set on the server.</returns>
+        [Obsolete("Please use Count")]
         public override long TotalCount
         {
             get

```

Comment:
I think you should change this getter to
```c#
[Obsolete(...)]
public override long TotalCount
{
  get
  {
    return Count;
  }
}
```

Reply:
fixed